tests/test_files/project/CALC-DATA.cbl

```diff
@@ -0,0 +1 @@
+s
```

Comment:
why is this one empty?

Reply:
This file was created automatically in tests. Should better cleanup those files.